<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="apply_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>361</width>
     <height>442</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="outer_button">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select Center Object</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Current Center:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="center_objs">
          <property name="text">
           <string>None</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="radius_box"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Shape:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="center_button">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Rotate objects towards center</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>How many times to duplicate?</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="circle_check">
        <property name="text">
         <string>Circle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="sphere_checkbox">
        <property name="text">
         <string>Sphere</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Rotate</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="0">
       <widget class="QCheckBox" name="random_outline_checkbox">
        <property name="text">
         <string>Random outline </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="uniform_checkbox">
        <property name="text">
         <string>Uniform outline</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="random_fill_checkbox">
        <property name="text">
         <string>Random fill</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Scatter type:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Duplicate object</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Radius in cm</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Select outer object(s)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Current Outer Objects:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="outer_objs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="warnings">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Outline/Fill tool for positioning multiple objects around or </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>surrounding another one at the center.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
